lib/bank/ach/file_parser/xck_entry.ex

```diff
@@ -0,0 +1,76 @@
+defmodule Bank.Ach.FileParser.XckEntry do
+  import NimbleParsec
+  import Bank.Ach.FileParser.Types
+  import Bank.Ach.FileParser.Common
+
+  @spec record() :: NimbleParsec.t()
+  def record() do
+    record_type_code("6")
+    |> concat(transaction_code())
+    |> concat(rdfi_identification())
```

Comment:
Let's use the same field names as the spec doc: `receiving_dfi_identification` unless there is a "/" in the name. I found one that was "RECEIVING COMPANY NAME/ID NUMBER" In those cases, let's see what moov does and go from there.

Reply:
got it. I'll make that change